tool/bots.sh

```diff
@@ -12,7 +12,7 @@ if [[ $RUNNER_OS == "Windows" ]]; then
     echo Installing Google Chrome Stable...
     # Install Chrome via Chocolatey while `addons: chrome` doesn't seem to work on Windows yet
     # https://travis-ci.community/t/installing-google-chrome-stable-but-i-cant-find-it-anywhere/2118
-    choco install googlechrome --acceptlicense --yes --no-progress --ignore-checksums
+    choco install googlechrome --acceptlicense --yes --no-progress --ignore-checksums --version 102.0.5005.115
```

Comment:
how often will we need to update this version number? Can you add a comment describing what the update process will look like for this?

Reply:
Good questions! I currently don't have an answer for the first one, but the update process would be manually updating the version number from https://community.chocolatey.org/packages/GoogleChrome#versionhistory, as dependabot doesn't support updating chocolatey pins.

With that in mind, we can hold off on moving this forward until I've got a better answer for you regarding how this would be updated.